FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,134 @@
+        <blockquote><i>NOTE: Godot 4.2 does not currently support runtime lightmap baking or UV2 unwrapping in exported builds. For now, consider using a different lighting solution for runtime building.<br></i></blockquote>
+        An example of runtime map building is as follows:</p>
+        <pre><code>extends Node
+
+@onready var func_godot_map := $FuncGodotMap as FuncGodotMap
+
+# We can connect the various build signals of the FuncGodotMap 
+# in order to add more functionality after building.
+func _ready():
+    func_godot_map.connect("build_complete", _build_complete)
+    func_godot_map.connect("build_failed", _build_failed)
+    func_godot_map.connect("unwrap_uv2_complete", _unwrap_uv2_complete)
+    func_godot_map.local_map_file = "res://maps/runtime_building.map"
+    func_godot_map.verify_and_build()
+
+# We need to wait for the build to finish before unwrapping mesh UV2s for lightmap baking.
+# Currently lightmap baking cannot be performed at runtime, so it's not terribly useful yet.
+# Consider creating a procedural Voxel GI solid class entity that can be baked at runtime instead.
+func _build_complete() -> void:
+    print("Success! Unwrapping UV2...")
+    func_godot_map.unwrap_uv2()
+
+func _build_failed() -> void:
+    printerr("Failed to build the map file! :(")
+
+func _unwrap_uv2_complete() -> void:
+    print("UV2 unwrapping completed!")</code></pre>
+    <p>An example of this can be found in <a href="https://github.com/func-godot/func_godot_runtime_building" target="_blank">func_godot_runtime_building repository</a>.</p>
+
+    <h1>Building from Map Editors</h1>
+    <p>
+        Much like idTech engine games, it's possible to build and launch directly into a map from NetRadiant Custom and TrenchBroom alike.
+        This can be quite a helpful tool for mappers to quickly try out their creations without a lengthy startup or debug menu process.
+    </p>
+
+    <h2>Godot project setup example</h2>
+
+    <p>
+        Regardless of which map editor you use, you'll need to check for any 
+        <a href="https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-method-get-cmdline-user-args">command line arguments.</a>
+
+        <p align="center"><img src="../images/tips_editorbuild_1.png"><br></p>
+
+        A basic example might look something like this:
+
+    </p>
+
+    <code><pre>
+# Somewhere in a script called when your project opens. Make sure you have a 
+# valid reference to a FuncGodot map or create one and apply your map settings to it.
+
+var func_godot_map: FuncGodotMap
+
+var map_override: String
+var cmd_line_args := OS.get_cmdline_user_args()
+
+for arg in cmd_line_args:
+    # Example argument, with a known delimiter.
+    if arg.contains("launch_debug::"):
+        var split := arg.split("::")
```

Comment:
I'd probably rename this from split to something more verbose and descriptive of what this variable's actual purpose is rather than what we've done to the argument. Maybe
```
for cmd in cmd_line_args:
    var cmd_args : PackedStringArray = cmd.split("::")
```
Since basically you're parsing the commands in `cmd_line_args` and splitting the command to retrieve the command's arguments that you've passed.

Reply:
The problem with cmd_args is that it's a parameter name and a parameter value. Additionally the for loop above isn't accurate as it's each cmd line argument, not each command. I've used the variable name `param_and_value` but it's a bit wordy, but says what it is on the tin.

Happy to hear other ideas what it should be called